compiler-plugin/src/main/scala/com/trueaccord/scalapb/compiler/GrpcServicePrinter.scala

```diff
@@ -175,6 +175,34 @@ final class GrpcServicePrinter(service: ServiceDescriptor, override val params:
           |""")
   }
 
+  private[this] def serviceDescriptor(service: ServiceDescriptor) = {
+
+    val grpcServiceDescriptor = "_root_.io.grpc.ServiceDescriptor"
+
+    val schemaDescriptor: String =
+      s""".setSchemaDescriptor(new _root_.io.grpc.protobuf.ProtoFileDescriptorSupplier {
```

Comment:
Instead of generating an anonymous supplier class for each service, I prefer that there would be a single class named `ConcreteProtoFileDescriptorSupplier` with a constructor that takes the descriptor and just return it. 

in scalapb-runt-ime-grpc